src/emsesp.cpp

```diff
@@ -343,17 +343,14 @@ void EMSESP::dump_all_entities(uuid::console::Shell & shell) {
         for (const auto & device : device_library_) {
             if (device_class.first == device.device_type) {
                 uint8_t device_id = 0;
-                // Mixer class looks at device_id to determine type and the tag
-                // so fixing to 0x28 which will give all the settings except flowSetTemp
-                if (device.device_type == DeviceType::MIXER) {
```

Comment:
the entities dump now has a dv.tag value of -31 (not sure why yet), so it breaks the modbus gen files. Any idea how to fix this @MichaelDvP ?